tests/lib/rules/require-computed-macros.js

```diff
@@ -287,5 +287,56 @@ ruleTester.run('require-computed-macros', rule, {
       output: "class Test { @computed.mapBy('children', 'age') someProp }",
       errors: [{ message: ERROR_MESSAGE_MAP_BY, type: 'MethodDefinition' }],
     },
+
+    // Self Referential
+    {
+      code: 'class Test { @computed() get foo() { return this.foo; } }',
+      options: [{ includeNativeGetters: true }],
+      output: 'class Test { @computed() get foo() { return this.foo; } }',
```

Comment:
We should use `output: null` to indicate there's no autofix, it's more clear and concise.

Reply:
wouldn't that imply the rule was deleting code?

Reply:
No. It means there's no autofix.

And there's actually a lint rule to enforce this but unfortunately it's not working on this code because the array of test cases has `.map(addComputedImport)` on it.

https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/blob/master/docs/rules/prefer-output-null.md